cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -89,31 +90,33 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
           resultTable()
         }
         p("When a user has been deleted, it will no longer appear on the list provided by `SHOW USERS`.")
-        query("SHOW USERS", assertNodesShown("User", column = "user")) {
+        query("SHOW USERS", assertAllNodesShown("User", column = "user")) {
           resultTable()
         }
         p("This command is optionally idempotent, with the default behavior to throw an exception if the user does not exists. " +
           "Appending `IF EXISTS` to the command will ensure that no exception is thrown and nothing happens should the user not exist.")
-        query("DROP USER jake IF EXISTS", ResultAssertions( r => {
+        query("DROP USER jake IF EXISTS", ResultAssertions(r => {
           assertStats(r, systemUpdates = 0)
         })) {}
       }
     }
     section("Role Management", "administration-security-roles") {
-      initQueries("CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED")
+      initQueries("CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED",
+        "CREATE USER user1 SET PASSWORD 'abc'", "CREATE USER user2 SET PASSWORD 'abc'", "CREATE USER user3 SET PASSWORD 'abc'",
+        "CREATE ROLE role1", "CREATE ROLE role2")
       p("Roles can be created and managed using a set of Cypher administration commands executed against the `system` database.")
       p("include::role-management-syntax.asciidoc[]")
       section("Listing roles", "administration-security-roles-show") {
         p("Available roles can be seen using `SHOW ROLES`.")
-        query("SHOW ROLES", assertNodesShown("Role")) {
+        query("SHOW ROLES", assertAllNodesShown("Role", column = "role")) {
           p("This is the same command as `SHOW ALL ROLES`.")
           resultTable()
         }
         p("There are multiple versions of this command, the default being `SHOW ALL ROLES`. " +
           "To only show roles that are assigned to users, the command is `SHOW POPULATED ROLES`. " +
-          "To see which users are assigned to roles can `WITH USERS` be appended to the commands. " +
+          "To see which users are assigned to roles `WITH USERS` can be appended to the commands. " +
           "This will give one result row for each user, so if a role is assigned to two users then it will show up twice in the result.")
-        query("SHOW POPULATED ROLES WITH USERS", assertNodesShown("Role")) {
+        query("SHOW POPULATED ROLES WITH USERS", assertRolesShown()) {
```

Comment:
This seems to assert that no roles are assigned users? Would it not be nicer to demo this command after some users are assigned roles, so we can see some results in the table?

Reply:
I added some roles and users that I needed for my commands down in `Assign roles to users` but before that nothing was returned in the `SHOW ROLES` either... If we get the default user and roles then there will be one here :)